dcpy/connectors/edm/publishing.py

```diff
@@ -122,3 +129,11 @@ def get_zip(dataset, version, filepath):
     stream = s3.get_file_as_stream(BUCKET, f"{dataset}/{version}/{filepath}")
     zip = ZipFile(stream)
     return zip
+
+
+if __name__ == "__main__":
+    if len(sys.argv) < 6:
+        raise Exception("Not enough arguments")
+    if sys.argv[1] != "publish":
+        raise Exception("Only 'publish' command currently implemented")
+    publish(sys.argv[2], sys.argv[3], sys.argv[4], sys.argv[5])
```

Comment:
This is pretty crude for now, but just wanted a simple way to call `publish` from github actions